Source/Modules/lang.cxx

```diff
@@ -3228,11 +3243,13 @@ Node *Language::symbolLookup(String *s, const_String_or_char_ptr scope) {
  * Tries to locate a class from a type definition
  * ----------------------------------------------------------------------------- */
 
-Node *Language::classLookup(const SwigType *s) const {
+Node *Language::classLookup(const SwigType *s) {
+  static Hash *classtypes = 0;
```

Comment:
See 0d4f0683be2cadc04de867f8e1e0a95ac9530a94 for the explanation.

And a static method can't be const.